custom/icds_reports/models/aggregate.py

```diff
@@ -1586,6 +1586,16 @@ class AggGovernanceDashboard(models.Model, AggregateMixin):
     total_3_6_male_benefit_in_month = models.IntegerField(null=True)
     total_3_6_female_reg_in_month = models.IntegerField(null=True)
     total_3_6_male_reg_in_month = models.IntegerField(null=True)
+    cbe_conducted_1 = models.TextField(null=True)
```

Comment:
why are you using `Textfield` instead of the more common `SmallIntegerField` 0/1 convention?

Reply:
its just 'yes'/'no' so that I dont need to change it to string while fetching

Reply:
I think we normally convert to numbers for efficiency reasons? not sure how important that is, maybe @calellowitz would know